llama_stack/providers/utils/sqlstore/sqlstore.py

```diff
@@ -48,20 +48,20 @@ def engine_str(self) -> str:
 
     @classmethod
     def sample_run_config(cls, __distro_dir__: str, db_name: str = "sqlstore.db"):
-        return cls(
```

Comment:
ah I hadn't realize we used non-dicts here. all `sample_run_config()` elsewhere always return dicts.